### DISCUSSION TITLE
Can't build an exe with github

### DISCUSSION THREAD

**Discussion by Andre0be** [rustdesk/rustdesk#7330] [Q&A]
Hello,

I have followed these instructions:

https://rustdesk.com/docs/en/dev/build/all/

I have made an import. Customized my server and public key and started the nightly build. Under "

Install flutter rust bridge deps"

However, I get the following error

`2024/03/06 22:09:15 [WARN] Skipping unresolvable module inline (tried \\?\D:\a\rustdesk\rustdesk\src\inline.rs, \\?\D:\a\rustdesk\rustdesk\src\inline\mod.rs, \\?\D:\a\rustdesk\rustdesk\src\ui\inline.rs, \\?\D:\a\rustdesk\rustdesk\src\ui\inline\mod.rs, )
2024/03/06 22:09:15 [WARN] Skipping unresolvable module version (tried \\?\D:\a\rustdesk\rustdesk\src\version.rs, \\?\D:\a\rustdesk\rustdesk\src\version\mod.rs, \\?\D:\a\rustdesk\rustdesk\src\lib\version.rs, \\?\D:\a\rustdesk\rustdesk\src\lib\version\mod.rs, )
2024/03/06 22:09:15 [WARN] Skipping unresolvable module bridge_generated (tried \\?\D:\a\rustdesk\rustdesk\src\bridge_generated.rs, \\?\D:\a\rustdesk\rustdesk\src\bridge_generated\mod.rs, \\?\D:\a\rustdesk\rustdesk\src\lib\bridge_generated.rs, \\?\D:\a\rustdesk\rustdesk\src\lib\bridge_generated\mod.rs, )
2024/03/06 22:09:15 [INFO] Phase: Transform IR
2024/03/06 22:09:15 [INFO] Phase: Generate Rust code
2024/03/06 22:09:16 [INFO] Phase: Generate Dart code
Flutter 3.10.6 • channel stable • https://github.com/flutter/flutter.git
Framework • revision f468f3366c (8 months ago) • 2023-07-12 15:19:05 -0700
Engine • revision cdbeda788a
Tools • Dart 3.0.6 • DevTools 2.23.1
2024/03/06 22:09:17 [INFO] Phase: Generating Dart bindings for Rust
Cannot open file, path = '.dart_tool\package_config.json' (OS Error: The system cannot find the path specified.
, errno = 3)
2024/03/06 22:09:25 [WARN] command="powershell" "-noprofile" "-command" "& \"flutter\" \"pub\" \"run\" \"ffigen\" \"--config\" \"C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\.tmpUeZIKq\"" stdout= stderr=
Error: /06 22:09:25 [ERROR] fatal: ffigen failed:
stderr: 
stdout: 
Error: Process completed with exit code 1.`

![Screenshot 2024-03-06 231904](https://github.com/rustdesk/rustdesk/assets/123382170/9cab23c1-f87d-417f-bbef-552b2dfc97d5)

What am I doing wrong?

**Comment by Andre0be** [comment]
Why is the support so bad?

**Reply by devdevfork** [reply]
its pretty bad yes.
cannot build any nightly builds for windows.

**Comment by Andre0be** [comment]
Rustdesk has real potential. But not like this.

I'm trying to build my own exe all the time. Nothing works. I would like to have my server and the public key stored directly in the exe.

**Reply by khushaal-nandwani** [reply]
hey were you able to do? I am trying to do the exact same thing. just started

**Comment by rustdesk** [comment]
#763
